docs/composables/useContentExamplesCode.ts

```diff
@@ -0,0 +1,27 @@
+const useContentExamplesCodeState = () => useState('components-meta', () => ({}))
```

Comment:
Shouldn't the key be different here? Otherwise it'll use the state of `useComponentMeta`.